Resources/Prototypes/Entities/Markers/atmos_blocker.yml

```
@@ -113,11 +113,20 @@
+  parent: MarkerBase
   components:
-  - type: AtmosFixMarker
-    gasMix:
-      volume: 2500
-      moles:
-        - 0 # oxygen
-        - 18710.71051 # nitrogen
-      temperature: 72
+    - type: Sprite
+      layers:
+        - sprite: Markers/atmos.rsi # {
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)